install/v2plugin/Dockerfile

```diff
@@ -1,14 +1,26 @@
+##
+#Copyright 2017 Cisco Systems Inc. All rights reserved.
+#
+#Licensed under the Apache License, Version 2.0 (the "License");
+#you may not use this file except in compliance with the License.
+#You may obtain a copy of the License at
+#http://www.apache.org/licenses/LICENSE-2.0
+#
+#Unless required by applicable law or agreed to in writing, software
+#distributed under the License is distributed on an "AS IS" BASIS,
+#WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#See the License for the specific language governing permissions and
+#limitations under the License.
+##
+
 # Docker v2plugin container with OVS / netplugin / netmaster
 
-FROM alpine:3.5
+FROM alpine:3.6
 LABEL maintainer "Cisco Contiv (https://contiv.github.io)"
 
 RUN mkdir -p /run/docker/plugins /etc/openvswitch /var/run/contiv/log \
  && echo 'http://dl-cdn.alpinelinux.org/alpine/v3.4/main' >> /etc/apk/repositories \
- && apk --no-cache add openvswitch=2.5.0-r0 iptables ca-certificates openssl curl \
- && wget -q -O /etc/apk/keys/sgerrand.rsa.pub https://raw.githubusercontent.com/andyshinn/alpine-pkg-glibc/master/sgerrand.rsa.pub \
- && wget https://github.com/andyshinn/alpine-pkg-glibc/releases/download/2.23-r1/glibc-2.23-r1.apk \
- && apk --no-cache add glibc-2.23-r1.apk
```

Comment:
Just to make sure, glibc is not needed? If so, please ref. an issue in contiv/netplugin (create if it doesn't exist).

Reply:
v2plugin was installing glibc from a third party package before. The whole point of alpine is that it doesn't use, nor require glibc, for all the packages it provides.

The netplugin binaries were being built on ubuntu or on another host. Go does require a libc for some functions of the net packages. One way to have libc independent binaries is to compile those as static binaries (which don't include glibc).

This PR makes the following changes related to glibc and static binaries:
- makes the build.sh script build static binaries
- removes the glibc from the v2plugin Dockerfile and doesn't use it in the updated Dockerfile of the main Docker image

This last point is also mentioned here: https://github.com/contiv/netplugin/pull/977#issue-258457997